app/src/main/resources/openapi_etor.yaml

```diff
@@ -40,3 +55,15 @@ components:
         patientId:
           type: string
           example: MRN7465737865
+    UnauthorizedError:
+      description: API key is missing or invalid
```

Comment:
I'm thinking it's probably better to have a generic description here for the error schema, and a more specific one in under the responses. The description here could be `Unauthorized Request`, and the one above could be `Authentication failed due to invalid token or unknown organization`?